t/cobrand/zurich.t

```diff
@@ -1072,15 +1072,9 @@ $mech->log_out_ok;
 subtest 'users at the top level can be edited' => sub {
     $mech->log_in_ok( $superuser->email );
     $mech->get_ok('/admin/users/' . $superuser->id );
-    $mech->content_contains('name="flagged">');
```

Comment:
Ah, so now upon looking more closely - Zurich, which as we chatted about is different in a few ways, uses 'flagged' on a user as a reference for a "government-internal" user (see the templates/web/zurich/admin/users/form.html template). So they need to maintain this behaviour, so these tests should pass unchanged.